Automattic/jetpack
# Fix support for allowed HTML tags in Subscribe block submit button

Fixes https://github.com/Automattic/jetpack/issues/25104

#### Changes proposed in this Pull Request:

* The previous fix made to the submit button was leaving all the HTML tags escaped, so it broke the HTML tags that are allowed to use for the submit button: `em`, `strong`, `s`, and `br`. This PR fixes support for allowed HTML tags in Subscribe block submit button. 

#### Other information:


#### Jetpack product discussion



N/A

#### Does this pull request change what data or activity we track or use?



No

#### Testing instructions:






1. Open a New Post editor
2. Add Subscribe block
3. Use one or more double-quotes in the button, use allowed HTML tags including line break, use `>` and `<` characters
4. Save the post
5. Open post in the frontend
6. All double-quotes should be displayed, line break should be preserved, `>` and `<` characters should be displayed, as well as custom HTML tags: `em`, `strong`, `s`, and `br`.

Editor:

<img width="653" alt="Screen Shot 2022-07-18 at 11 13 36" src="https://user-images.githubusercontent.com/727413/179480670-66c38943-71c2-483c-a574-a677ce498026.png">

Frontend:

<img width="618" alt="Screen Shot 2022-07-18 at 11 13 45" src="https://user-images.githubusercontent.com/727413/179480932-bbd6570d-0aff-4537-b824-3b6238d83d8c.png">
